install-dependencies.sh

```diff
@@ -103,6 +103,12 @@ fedora_python3_packages=(
     python3-setuptools
     python3-psutil
     python3-distro
+    python3-six
+    python3-geomet-0.2.1-5.fc33
```

Comment:
Dequeued, I just noticed this - it won't work when we upgrade to Fedora 34, and the arm toolchain I built already didn't work, the build process upgraded it.

Is geomet=0.2.1 needed for cassandra-driver?  Or will it work with a later version?

Reply:
I see it prohibts 0.3:

https://github.com/datastax/python-driver/commit/892340fad6438f270ff8183ec72603340aacce2c


I think we should use our cassandra driver, not datastax, and fix it to be able to work with 0.3.

Reply:
It fails on x86 too, since install-dependencies.sh does a dnf update somewhere.